### PR TITLE
Use with in examples, remove close in json()

### LINKED ISSUE #186: Please re-write examples to use `with ... as request:` style
The examples, at least simpletest, use code structured like this:
```py
print("Fetching text from %s" % TEXT_URL)
response = requests.get(TEXT_URL)
print("-" * 40)

print("Text Response: ", response.text)
print("-" * 40)
response.close()
```
In modern Python, using a `with` statement is idomatic and provides the best resource management, because the need to manually call close (including in exception cases) is removed:
```py
print("Fetching text from %s" % TEXT_URL)
with requests.get(TEXT_URL) as response:
    print("-" * 40)
    print("Text Response: ", response.text)
    print("-" * 40)
```

### PR DESCRIPTION
**Pull request by justmobilize**
Update the examples to use `with request.method as response:` and removed the close in the `Response.json()` as it doesn't match CPython

Tested the wifi standard and advanced and the rocketlive in advanced

fixes #186

### CODE REVIEW

File: adafruit_requests.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def json(self) -> Any:
         obj = json_module.load(self._raw)
         if not self._cached:
             self._cached = obj
-        self.close()
+
```
Comment (justmobilize) [inline adafruit_requests.py:324]: Removed the `.close()` here, as it's not standard in CPython

File: examples/wifi/expanded/requests_wifi_api_fitbit.py [hunk 2]
```diff
@@ -252,9 +251,9 @@ def time_calc(input_time):
             print(" | Attempting to GET Fitbit JSON!")
             FBIS = FITBIT_INTRADAY_SOURCE
             FBH = fitbit_header
-            fitbit_get_response = requests.get(url=FBIS, headers=FBH)
             try:
-                fitbit_json = fitbit_get_response.json()
+                with requests.get(url=FBIS, headers=FBH) as fitbit_get_response:
```
Comment (justmobilize) [inline examples/wifi/expanded/requests_wifi_api_fitbit.py:255]: A few of these, I moved the initial connection into the try/except block

File: examples/wifi/expanded/requests_wifi_api_github.py [hunk 3]
```diff
@@ -92,9 +94,6 @@ def time_calc(input_time):
             print("Full API GET URL: ", GITHUB_SOURCE)
             print(github_json)
 
-        github_response.close()
-        print("✂️ Disconnected from Github API")
```
Comment (justmobilize) [inline examples/wifi/expanded/requests_wifi_api_github.py:96]: Removed these closes, and the disconnected message

**Comment by DJDevon3** [comment]
Wow, thank you.  Saved me from all these changes. 🙏

**Review by FoamyGuy — APPROVED** [review]
Looks good to me. Thanks for working through all of those!

I tested a handful of them on a Feather S3 TFT and the ESP32SPI ones on a PyPortal Titano

**Pull request merged**